src/main.tsx

```diff
@@ -21,6 +21,17 @@ import { ConfigWrapper } from "./Config.tsx";
 import "./index.scss";
 import { AppStateWrapper } from "./AppState.tsx";
 
+if('serviceWorker' in navigator){
```

Comment:
nit: align formatting to rest of the file

Reply:
Addressed in recent PR